git_machete/client.py

```diff
@@ -1768,11 +1768,6 @@ def create_github_pr(self, head: str, draft: bool) -> None:
             return
 
```

Comment:
Hmmm why is the `except MacheteException as e:` needed above? Shouldn't a `MacheteException` coming from `__sync_before_creating_pr` be caught in top-level handler (`def launch`) as any other `MacheteException`?